logger-service/logger-service.js

```diff
@@ -0,0 +1,82 @@
+const fsExtra = require('fs-extra');
+const outputCache = new Map();
+
+const AppenderTypes = {
+	NONE: 'none',
+	NORMAL: 'normal',
+	STRUCTURED: 'structured'
+};
+
+const outputParameterStoragePath = `${process.cwd()}/.outputParameter`;
+
+class Logger {
+
+	constructor() {
+	}
+
+	log() {
+		const appenderType = this.getOutputParameterValue();
+		const args = Array.prototype.slice.call(arguments);
+
+		if (appenderType === AppenderTypes.NONE) {
+			return;
+		}
+
+		if (appenderType === AppenderTypes.STRUCTURED) {
+			this.logToFile(args);
+			return;
+		}
+
+		this.logToConsole(args);
+	}
+
+	logToConsole(args) {
+		console.log.apply(console, args);
```

Comment:
I think you can use the spread operator here
http://es6-features.org/#SpreadOperator